code/game/objects/structures/crates_lockers/closets/job_closets.dm

```diff
@@ -58,47 +58,6 @@
 	new /obj/item/clothing/head/surgery/hairnet(src)
 	new /obj/item/clothing/head/surgery/hairnet(src)
 
-/*
- * Custodian
- */
-/obj/structure/closet/jcloset
```

Comment:
If you want to rename that, you also need to replace all existing instances of it on all maps.

Reply:
fixed.